test/utils/feature.flag.spec.unit.ts

```diff
@@ -0,0 +1,54 @@
+import { isActiveFeature } from "../../src/utils/feature.flag";
+
+describe("feature flag tests", () => {
+
+  it("should return false if variable is 'false'", () => {
+    const active = isActiveFeature("false");
+    expect(active).toBeFalsy();
+  });
+
+  it("should return false if variable is '0'", () => {
+    const active = isActiveFeature("0");
+    expect(active).toBeFalsy();
+  });
+
+  it("should return false if variable is ''", () => {
+    const active = isActiveFeature("");
+    expect(active).toBeFalsy();
+  });
+
+  it("should return false if variable is undefined", () => {
+    const active = isActiveFeature(undefined);
+    expect(active).toBeFalsy();
+  });
+
+  it("should return false if variable is 'off'", () => {
+    const active = isActiveFeature("off");
+    expect(active).toBeFalsy();
+  });
+
+  it("should return false if variable is 'OFF'", () => {
+    const active = isActiveFeature("OFF");
+    expect(active).toBeFalsy();
+  });
+
+  it("should return true if variable is 'on'", () => {
```

Comment:
What happens if a random string is entered?

Reply:
just added a new test for this.